beacon-chain/rpc/prysm/v1alpha1/validator/proposer.go

```diff
@@ -248,9 +253,16 @@ func (vs *Server) proposeGenericBeaconBlock(ctx context.Context, blk interfaces.
 		return nil, fmt.Errorf("could not tree hash block: %v", err)
 	}
 
-	blk, err = vs.unblindBuilderBlock(ctx, blk)
-	if err != nil {
-		return nil, err
+	if slots.ToEpoch(blk.Block().Slot()) >= params.BeaconConfig().CapellaForkEpoch {
```

Comment:
dont we have a safer helper to check if epoch is capella or greater ? i imagine this conditional exists elsewhere as well

Reply:
I dont think so. Yes, similar conditions exist in many places. We can replace them with helper in subsequent PRs